code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -499,7 +499,7 @@
 			visible_message(span_warning("[helper] tries to pat [src] on the head, but can't reach!"))
 			return
 		else if(HAS_TRAIT(src, TRAIT_QUICKREFLEXES) && (src.stat != UNCONSCIOUS) && (!src.handcuffed))
-			visible_message(span_warning("[helper] tries to pat [src] on the head, but their hand is swatted away!"))
+			to_chat(span_notice("You try to pat [p_them()] on the head, but they move their head out of the way."))
```

Comment:
this will fail and send to nobody

also switch this back to the visible message, this is a v isible action

Reply:
fguck yu duoeude